koji-prometheus-exporter.py

```diff
@@ -222,6 +223,19 @@ def koji_enabled_hosts_capacity(hosts):
         yield sum([h['capacity'] for h in hosts[channel]]), [channel]
 
 
+def koji_hosts_last_update(hosts):
+    koji_builders = {}
+    for value in hosts:
```

Comment:
This was a little tricky to read for me at first.  Is `hosts` a dict?

Perhaps simplify the two nested loops down to one with:

```
for channel, host in hosts.items():
    host['channel'] = channel
    koji_builders[host['name']] = host
for hostname, host in koji_builders.items():
    yield getLastHostUpdate(host, ts=True), [hostname, host['channel']]
```

WDYT?